### PR TITLE
[travis] ci updates

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_map_accessibility_analysis/CMakeLists.txt [hunk 1]
```diff
@@ -1,28 +1,20 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_map_accessibility_analysis)
 
-set(catkin_RUN_PACKAGES
```
Comment (ipa-rmb) [inline cob_map_accessibility_analysis/CMakeLists.txt:4]: Why did you remove this handy trick to avoid listing all dependencies twice in this file? Is there any good reason why we should do this e.g. in the vision packages as well or is it just preferred style?

Comment (fmessmer) [reply cob_map_accessibility_analysis/CMakeLists.txt:4]: like this, the packages are not alphabetically sorted - i.e. `message_generation`/`message_runtime` - which is required by `catkin_lint`

Comment (ipa-rmb) [reply cob_map_accessibility_analysis/CMakeLists.txt:4]: Ah, I see. Thanks for the explanantion.

**Comment by fmessmer** [comment]
speed-up:
 - before: Ran for 1 hr 1 min 3 sec 
 - after: Ran for 16 min 40 sec
